### ISSUE TITLE
Document jQuery.event.special.<event> structure

### ISSUE THREAD

**Issue by markelog** [jquery/api.jquery.com#752]
Example - https://github.com/jquery/jquery/blob/0019a463bdcb81dc6ba3434505a45774ca27f363/src/event.js#L553-602

**Comment by dmethvin** [comment]
It's here at the moment: http://learn.jquery.com/events/event-extensions/

**Comment by dmethvin** [comment]
`preDispatch` and `postDispatch` aren't there, we discussed removing in https://github.com/jquery/jquery/pull/2358

**Comment by markelog** [comment]
Yes, this ticket specifically about those functions - https://github.com/jquery/jquery/pull/2358#issuecomment-107633332 :-)

**Comment by dmethvin** [comment]
Ok, then I am not sure if the subject above describes what needs to be done.

**Comment by markelog** [comment]
Comment and title changed, looks better?

**Comment by dmethvin** [comment]
Yes! Now I know what you meant. :smiley_cat:
